Description: fix fonts
 default font have changed, replaced by another one
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/font.cpp
+++ b/font.cpp
@@ -72,7 +72,7 @@
 	dpy = XOpenDisplay( NULL );
 
 	/* Get the font information */
-	fontInfo = XLoadQueryFont( dpy, "-adobe-helvetica-medium-r-normal--32-*-*-*-p-*-iso8859-1" );
+	fontInfo = XLoadQueryFont( dpy, "-bitstream-bitstream charter-medium-r-normal--32-*-*-*-p-*-adobe-standard" );
 
 	/* If the above font didn't exist try one that should */
 	if ( fontInfo == NULL )
